### ISSUE TITLE
TrackingMeasures: order of tabs

### ISSUE THREAD

**Issue by strambc** [OmegaProject/Omega#190]
All Tracking Measures module should have the following TAB order:

Browser -> Run -> Graph -> Results
TMI, TMM and TMD need to be corrected

TMV is already correct

**Comment by Akramhar** [comment]
should be solved in omega-0.27

**Comment by strambc** [comment]
TMI is still the wrong way

**Comment by Akramhar** [comment]
should be fixed in omega-0.29
